var/spack/repos/builtin/packages/cbtf-argonavis/package.py

```diff
@@ -67,6 +63,40 @@ class CbtfArgonavis(Package):
 
     parallel = False
 
+    build_directory = 'build_cbtf_argonavis'
+
+    def build_type(self):
+        return 'None'
```

Comment:
Most CMake packages use `RelWithDebInfo`. I've never seen one actually use `None` before. What's the difference?